src/lib/ens-helpers.ts

```diff
@@ -1,17 +1,16 @@
+import { bytesToPacket } from "@ensdomains/ensjs/utils";
 import { type Hex, concat, keccak256, namehash, toHex } from "viem";
 
-// TODO: pull from ens utils lib or something
 export const NAMEHASH_ZERO = namehash("");
 export const NAMEHASH_ETH = namehash("eth");
 
-// TODO: this should probably be a part of some ens util lib
+// TODO: this should probably be a part of some ens util lib but isn't exposed by viem or ensjs
 export const makeSubnodeNamehash = (node: Hex, label: Hex) => keccak256(concat([node, label]));
```

Comment:
I put a bunch of work into drafting a library for these kinds of operations this summer. It's somewhere on my laptop. Please feel welcome to assign a task for me to finish that library so that we can make use of it. It would be highly useful for our indexer and also for devs in general building on ENS :)